meta/classes/image_types.bbclass

```diff
@@ -137,12 +137,12 @@ UBI_VOLNAME ?= "${MACHINE}-rootfs"
 multiubi_mkfs() {
 	local mkubifs_args="$1"
 	local ubinize_args="$2"
-    
+
```

Comment:
Normally, I'm a fan of embedding whitespace fixups in my commits. But in the interest of making this commit as lightweight to carry out-of-stream as is possible, could you remove these unnecessary differences from the diff?